livekit-plugins/livekit-plugins-inworld/livekit/plugins/inworld/tts.py

```diff
@@ -0,0 +1,287 @@
+# Copyright 2025 LiveKit, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import asyncio
+import base64
+import os
+from collections.abc import AsyncGenerator
+from dataclasses import dataclass, replace
+from typing import Literal, cast
+
+from inworld_sdk import InworldClient, models
```

Comment:
i don't think we should import another SDK. often we see third-party SDKs not handling timeouts properly. and it can be a pain to work with.

if this is a standard HTTP API that produces data, we should just use that directly

Reply:
Lol, by third party its just me... I made the plugin, I was talking to Inworld to maybe pass control of it to them cause I assumed they'd want it, and looking at how we pass our own http clients into some sdks I made sure this was an option for this, but I'll roll in the logic into this plugin.